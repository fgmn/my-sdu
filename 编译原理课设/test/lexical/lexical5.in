2=3=4 =			a=
(-1



);