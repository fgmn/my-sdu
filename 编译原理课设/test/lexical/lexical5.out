NUMBER 2
=
NUMBER 3
=
NUMBER 4
=
IDENTIFIER A
=
(
-
NUMBER 1
)
;
.
